R/assertions.R

```diff
@@ -27,8 +27,18 @@ assert_experiment = function(e, .var.name = vname(e)) {
 #'   Set of allowed feature types.
 #' @param task_properties :: `character()`\cr
 #'   Set of required task properties.
+#' @param allow_empty_rows :: `logical()`\cr
```

Comment:
Is this the same as setting `min_rows` to 1?

Reply:
Yes, changed it :+1: